libgalois/include/katana/analytics/Utils.h

```diff
@@ -59,6 +61,76 @@ ConstructNodeProperties(
   return pg->AddNodeProperties(res_table.value());
 }
 
+inline std::shared_ptr<arrow::ChunkedArray>
+ApplyBitMask(
+    const std::shared_ptr<arrow::ChunkedArray>& chunked_array,
+    const uint8_t* bit_mask) {
```

Comment:
I really think we need a BitMask type. Passing them around untyped in this way is dangerous.

Reply:
I will try to figure out how to use the `Bitmap` class in the `arrow` library then. thanks

Reply:
No, that's not the point. You can pass `const NUMAArray<uint8_t>&` everywhere and use it as a pointer line 72 and

Reply:
I feel that the arrow's Bitmap class is more clean for this. So I have updated the code with that